pkgs/build-support/node/build-npm-package/hooks/default.nix

```diff
@@ -9,7 +9,7 @@
 , installShellFiles
 }:
 
-{
+rec {
```

Comment:
It'd be nice to avoid this... In general I feel the need to put all of these hooks straight in top-level, and not inside the `npmHooks` attribute set.